release_notes.md

```diff
@@ -1,21 +1,10 @@
 # What's New
 
-## Introducing Top SOC Questions
-
-Edit or disable existing prompts or add new prompts based on your security posture.
-
-- You can now customize your interactions with the LLM by providing specific details and guidelines for your chats. Define your SOC questions to customize your LLM interactions.
-
-- Specific modules now have pre-defined questions. For example, under the **Alerts** module, questions include:
-
-    - Get Alert Summary
-    - Generate Response Recommendations
-    - Generate Investigation Report
-    - MITRE ATT&CK Insight
-    
-    Similarly, for the **Incident** module, questions may be:
-    
-    - Get Incident Summary
-    - Generate Response Recommendations
-    - Generate Investigation Report
-    - MITRE ATT&CK Insight
+- The AI Assistant can understand and respond to context sensitive conversations and perform given SOC tasks
+    - Through enhanced comprehension, the bot can navigate to modules, its records, and playbook logs
+    - Sensitive data is being actively masked for all interactions with the API, considering the various modules to which the bot has access
+    - The bot can now generate playbooks for all applicable modules and now has access to create and update records
+    - The modes &mdash; Playbook Blocks and Conversation Mode &mdash; have been deprecated
+
+- The past conversations remain in memory and help provide context to the bot, even after the user has logged out.
```

Comment:
not in memory. The past conversations are retained until user clicks on clear conversation. More details can be found in solution pack doc where we have mentioned assistant api details